packages/flutter_tools/test/general.shard/base/build_test.dart

```diff
@@ -715,6 +715,39 @@ void main() {
       ]);
     }, overrides: contextOverrides);
 
+    testUsingContext('builds shared library for android-arm without dwarf stack traces due to empty string', () async {
```

Comment:
Is this the right test?

Reply:
Its checking that `splitDebugInfo: ''` doesn't introduce dwarf-stack-traces